node.gyp

```diff
@@ -13,6 +13,7 @@
     'node_use_bundled_v8%': 'true',
     'node_shared%': 'false',
     'force_dynamic_crt%': 0,
+    'ossfuzz' : 'true',
```

Comment:
```suggestion
    'ossfuzz' : 'false',
```
Functionally it doesn't really matter because it's always overridden by configure.py but it's good to have the default mirror the default from configure.py.

Reply:
argh that's completely my mistake. I definitely agree.